src/servers/grpc_server.cc

```diff
@@ -249,14 +252,13 @@ GRPCServer::~GRPCServer()
 
 Status
 GRPCServer::Create(
```

Comment:
Create with all 4 ports and put CreateUniqueEndpointPorts logic here.

Reply:
Will do